src/pages/Home/Home.page.jsx

```diff
@@ -1,37 +1,30 @@
 import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
-
-import { useAuth } from '../../providers/Auth';
 import './Home.styles.css';
 
+import videos from '../../mockData/youtube-videos-mock.json';
+import VideoCard from '../../components/VideoCard';
+
 function HomePage() {
-  const history = useHistory();
   const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
-
   return (
     <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
+      {videos.items.map((video, index) => {
+        console.log(index);
+        const { id, snippet } = video;
+        const { thumbnails, description, channelTitle, title, publishedAt } = snippet;
+        const img = thumbnails && thumbnails.high && thumbnails.high.url;
+        if (id && id.videoId)
+          return (
+            <VideoCard
```

Comment:
non-blocker, for any render related to map I think we should put key props into the component. for more information https://reactjs.org/docs/reconciliation.html#keys